test/workspace.js

```diff
@@ -249,4 +249,61 @@ describe('Workspace', function() {
       });
     });
   });
+
+  describe('Creating 2 projects from template', function() {
+    beforeEach(givenEmptySandbox);
```

Comment:
No, we don't need to `resetWorkspace` as I'm using `loadWorkspace`. Eventually, @bajtos suggest changing all the `resetWorkspace` to use `loadWorkspace`.